Model/DutyCalculator.php

```diff
@@ -577,11 +577,18 @@ protected function prepareRequest($freightCharge, $shippingAddress)
                 $itemData['skuNumber']=$item->getSku();
                 $itemData['itemValue']=['value'=>($item->getRowTotal() - $item->getDiscountAmount())/$item->getQty(),'currency'=>$quote->getQuoteCurrencyCode()];
                 $itemData['itemQuantity']=['value'=>$item->getQty(),'unit'=>"PCS"];
-                $itemData['countryOfOrigin'] = $this->getCountryOfOrigin($item->getSku());
-                if (!$itemData['countryOfOrigin']) {
-                    $itemData['countryOfOrigin'] = $request['senderAddress']['country'];
+
+                //country of origin from uploaded hs code file
+                $countryOfOrigin = $this->getCountryOfOrigin($item->getSku());
+                if (!$countryOfOrigin) {
+                    $countryOfOrigin = $this->reachHelper->getDefaultCountryOfOrigin();
+                }
+                //country of origin from default country of origin drop down
```

Comment:
small detail, this comment is in a weird place I would have expected it inside of the if statement (if it's needed at all, the code is fairly self-explanatory)

Reply:
commit 7f6806366f5b4417cbbe8f01d32b4507b9b24e06.
Thanks!